test/folders.test.js

```diff
@@ -0,0 +1,53 @@
+const folders = require('../src/folders')
```

Comment:
These tests provide good coverage for the basic CRUD operations. For future enhancements, consider making the tests more independent by creating and deleting test folders within `beforeEach/afterEach` or `beforeAll/afterAll` blocks. This can help isolate failures if the test suite grows.